### PR TITLE
SDK-448: Add SSL support for Docker container

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ services:
     links:
       - joomladb:mysql
     ports:
-      - "8080:80"
+      - "6001:443"
```
Comment (bucky-boy) [inline docker-compose.yml]: Why change the port?  Can't we just have all the examples configured to run on the same port?

File: yoti/admin/views/yoti/tmpl/default.php [hunk 2]
```diff
@@ -18,6 +18,9 @@
                 <?php echo JText::_('Note: On the Yoti Dashboard the callback URL should be set to:'); ?>
                 <strong><?php echo JUri::root(false).'index.php?option=com_yoti&task=login'; ?></strong>
             </p>
+            <p>
+                <?php echo JText::_('Warning: User IDs provided by Yoti are only valid within the scope of the application. Changing your Yoti application will result in different Yoti user IDs.'); ?>
```
Comment (bucky-boy) [inline yoti/admin/views/yoti/tmpl/default.php]: How about:
"User IDs are unique to each Yoti Application.  Using a different Yoti Application means you will receive a different User ID for all of your users."

Comment (sidibos) [reply yoti/admin/views/yoti/tmpl/default.php]: I will use the same text here from Drupal PRs.

File: docker-compose.yml (hunk 1 above)
Comment (sidibos) [reply docker-compose.yml]: They will end up colliding if run them simultaneously, that's why I am using a different port for each.

Comment (bucky-boy) [reply docker-compose.yml]: ok

**Pull request merged**